repose-aggregator/components/filters/header-id-mapping/src/test/scala/org/openrepose/filters/headeridmapping/HeaderIdMappingFilterTest.scala

```diff
@@ -0,0 +1,268 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2016 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+
+package org.openrepose.filters.headeridmapping
+
+import javax.servlet.{ServletResponse, FilterChain}
+
+import org.junit.runner.RunWith
+import org.mockito.ArgumentCaptor
+import org.mockito.Matchers._
+import org.mockito.Mockito._
+import org.openrepose.commons.utils.http.PowerApiHeader
+import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper
+import org.openrepose.filters.headeridmapping.config.{HttpHeader, HttpHeaderList, HeaderIdMappingConfig}
+import org.scalatest.{Matchers, BeforeAndAfter, FunSpec}
+import org.scalatest.junit.JUnitRunner
+import org.springframework.mock.web.MockHttpServletRequest
+
+import scala.collection.JavaConverters._
+import scala.language.implicitConversions
+
+@RunWith(classOf[JUnitRunner])
+class HeaderIdMappingFilterTest extends FunSpec with BeforeAndAfter with Matchers {
+
+  import HeaderIdMappingFilterTest._
+  import PowerApiHeader._
+
+  var filter: HeaderIdMappingFilter = _
+  var servletRequest: MockHttpServletRequest = _
+  var servletResponse: ServletResponse = _
+  var filterChain: FilterChain = _
+
+  before {
+    servletRequest = new MockHttpServletRequest
+    servletResponse = mock(classOf[ServletResponse])
+    filterChain = mock(classOf[FilterChain])
```

Comment:
I totally missed it, but you could clean this up a tiny bit by mixing in the MockitoSugar trait.